db/seeds.rb

```diff
@@ -61,7 +64,8 @@
   reason_to_donate: 'I want to support a female president',
   slug: 'hillary-clinton',
   info_url: 'https://www.hillaryclinton.com/',
-  donation_url: 'https://www.hillaryclinton.com/donate/'
+  donation_url: 'https://www.hillaryclinton.com/donate/',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.